components/api/src/main/java/com/hotels/styx/api/HttpResponse.java

```diff
@@ -192,7 +192,7 @@ public LiveHttpResponse stream() {
      *
      * @return a set of cookies
      */
-    public Set<ResponseCookie> cookies() {
+    public List<ResponseCookie> cookies() {
```

Comment:
This is a breaking change in the api. Not sure what the implications would be.

Reply:
Well spotted!
This could break StyxPres or Vrbo plugins if they upgrade to use the latest version of Styx.
I will check all plugins and update them